src/docs/product/integrations/integration-platform/index.mdx

```diff
@@ -247,6 +247,20 @@ When you're ready for the publication process, click "Publish" next to the integ
 
 ![Buttons providing the options to publish or delete.](publish-button.png)
 
+## Image Requirements
+
+Published integrations are required to have a logo image uploaded. The requirements for those are fairly rigid:
+* Logo size must be between 256x256 px and 1024x1024px
+* The file must be a PNG
+* The background must be transparent (unless the logo takes up the entire space)
+
+If the integration has a [UI component](#ui-components), it must also upload an icon that follow these requirements:
+* Icon size must be between 256x256 px and 1024x1024px
+* The file must be a PNG in RGBA format
+* All pixels must be black or transparent
+
+Internal integrations can upload logos and icons if they wish, but they are not required.
```

Comment:
suggestion: "can have uploaded logos and icons, though they are not required"

The `can` and `if they wish` suggest the integration is doing the uploading/wishing